packages/jobs/lib/activities.ts

```diff
@@ -107,7 +97,8 @@ export async function scheduleAndRouteSync(args: ContinuousSyncArgs): Promise<bo
 
     // https://typescript.temporal.io/api/classes/activity.Context
     const context: Context = Context.current();
-    const syncType = (await initialSyncExists(syncId as string)) ? SyncType.INCREMENTAL : SyncType.INITIAL;
+    const lastSyncDate = await getLastSyncDate(syncId);
+    const syncType = lastSyncDate ? SyncType.INCREMENTAL : SyncType.FULL;
```

Comment:
Should it be full or just go back to being an `INITIAL`? Locally I'm also seeing inconsistent state between doing a full resync and sometimes it being a FULL and sometimes being an INITIAL. I feel like it is easier and less to manage to just have it go back to `INITIAL`

Reply:
yep that's a leftover from my early experimentation, good catch

Reply:
actually after changing to use FULL_RUN, might make more sense to stick with FULL.
INITIAL does not make a lot of sense when we think about it

Reply:
Changed ed3472d to INITIAL (so my prev comment is still relevant)